<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockRefine</class>
 <widget class="QWidget" name="StockRefine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="localisation_Label">
     <property name="text">
      <string>Localisation : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="localisation_LineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="state_Label">
     <property name="text">
      <string>State : </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="manufacturer_Label">
     <property name="text">
      <string>Manufacturer : </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="item_Label">
     <property name="text">
      <string>Item : </string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="item_LineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="manufacturer_LineEdit"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="refine_PushButton">
     <property name="text">
      <string>Refine</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="new_PushButton">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="underMinimum_CheckBox">
     <property name="text">
      <string>Under minimum</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="overMaximum_CheckBox">
     <property name="text">
      <string>Over maximum</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
